filebeat/module/logstash/log/_meta/fields.yml

```diff
@@ -9,7 +9,7 @@
       description: >
         The module or class where the event originate.
     - name: thread
-      type: text
+      type: keyword
```

Comment:
👍

Reply:
Changed to `keyword` with `text` multi-field in https://github.com/elastic/beats/pull/10417.